test/Sema/subclass_property_import.swift

```diff
@@ -0,0 +1,12 @@
+// RUN: rm -rf %t
+// RUN: mkdir -p %t
+// RUN: %target-swift-frontend -emit-module -o %t/rdar32973206_a.swiftmodule %S/Inputs/rdar32973206_a.swift
+// RUN: %target-swift-frontend -I %t -emit-module -o %t/rdar32973206_b.swiftmodule %S/Inputs/rdar32973206_b.swift
+// RUN: %target-swift-frontend -I %t -emit-module -o %t/rdar32973206_c.swiftmodule %S/Inputs/rdar32973206_c.swift
+// RUN: %target-swift-frontend -I %t -typecheck -verify %s
+
+import rdar32973206_a
+import rdar32973206_b
+import rdar32973206_c
+
+let _ = B.foo // Ok
```

Comment:
Can you add tests for referencing `A.foo` as well as adding a new `class C : B {}` and a reference through `C.foo`?

You can check that the right symbols are referenced by using `emit-sil` and `FileCheck` in a `RUN:` line.